core/src/main/scala/org/apache/spark/internal/config/ConfigBuilder.scala

```diff
@@ -199,6 +200,11 @@ private[spark] case class ConfigBuilder(key: String) {
     this
   }
 
+  def version(v: String): ConfigBuilder = {
+    _version = v
```

Comment:
Shall we add a format check for `v`?

Reply:
Although we can checked it, but I think convention is greater than configuration.